openedx/core/djangoapps/credentials/management/commands/notify_credentials.py

```diff
@@ -246,11 +253,20 @@ def send_notifications(self, certs, grades, site_config=None, delay=0, page_size
             )
 
             user = User.objects.get(id=grade.user_id)
+
+            # Grab mode/status from cert call
+            mode, status = None, None
+            if course_cert_info:
+                key = (user.id, str(grade.course_id))
+                if key in course_cert_info:
+                    mode = course_cert_info[key].get('mode', None)
+                    status = course_cert_info[key].get('status', None)
```

Comment:
I believe we can simplify it:
```suggestion
            key = (user.id, str(grade.course_id))
            cert_info = course_cert_info.get(key, {})
            mode = cert_info.get('mode', None)
            status = cert_info.get('status', None)
```

Reply:
Done